src/cmd-run

```diff
@@ -394,6 +399,21 @@ set -- -name coreos -m "${VM_MEMORY}" -nographic \
               -object rng-random,filename=/dev/urandom,id=rng0 -device virtio-rng-"${devtype}",rng=rng0 \
               "$@"
 
+# We want to strongly emphasize use of TPM devices in CoreOS, so let's provision
+# one by default.  This would be nicer if we didn't need to pass a persistent
+# state directory to swtpm, and we could more conveniently "lifecycle bind"
+# the swtpm and qemu processes.
+if [ "${SWTPM}" = 1 ] && [ -x /usr/bin/swtpm ]; then
```

Comment:
This requires adding `swtpm` to to the list of deps, no?

I'm not sure about doing this by default though. I see `cosa run` as being a lightweight `exec qemu` wrapper and then opt-in switches for things we want to test (e.g. `--ssh`, `--uefi-secure`, `--srv`, etc...). Why does this not fall in the same category?

Reply:
Yep, will do a PR to add it.

I'd argue that there's no reason not to do a vTPM by default.  I'd like to change the perception closer to a situation where *not* having a TPM is the odd case.

Compared with other flags:

 - `--ssh` and `--uefi-secure`: Behavioral change, whereas this just *adds* functionality without breaking anything else
 - `--srv`: Requires an option today, though I guess in theory we could default to the builddir?  Might be useful for some people.

Reply:
OK gotcha, SGTM (assuming we can fix the filesystem leak!)